masterlist.yaml

```
@@ -853,10 +854,46 @@ plugins:
       - 'BOM_pathgrid_reset.esp'
       - 'multipatch.esp'
 
+   ##Optional files##
```

The subsection is fine, I'd append the mod name or an abbreviation before it though.
And I'd try and keep files from the same mod together, replacers can be kept separate.

```suggestion
   ### BCOM - Optional files ###
```